caf-audit/src/test/java/com/hpe/caf/auditing/elastic/ElasticAuditIT.java

```diff
@@ -609,4 +603,87 @@ private static void deleteDocument(TransportClient client, String indexId, Strin
         }
     }
 
+    private class AuditEvent {
```

Comment:
In the latest commit I have adjusted the name to TestAuditEvent and I've adjusted the names of the type variables and their return methods. Added a comment to the TestAuditEvent to make the reader aware of its purpose.